Tests/Unit/Service/RenderingTest.php

```diff
@@ -41,6 +41,19 @@ class Tx_HappyFeet_Tests_Unit_Service_RenderingTest extends PHPUnit_Framework_Te
      */
     public function setUp()
     {
+        $GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['extbase_object'] = array(
```

Comment:
If we are configuring this block for unit tests then in my opinion its not really a unit test.